release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,53 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	rm -f artifacts.zip >/dev/null 2>&1
+	if ! wget --quiet --ca-cert=/etc/ssl/certs/ISRG_Root_X1.pem "$artifactsUrl" ; then
+		echo "Failed to download artifacts from $artifactsUrl" 1>&2
+		return 1
+	fi
+ 	rm -r artifacts >/dev/null 2>&1
+ 	if test -e artifacts ; then
+ 	    rm -rf artifacts
```

Comment:
We should repeat the previous failed command (to get the error message). If nothing else, this helps the reader to understand why there are two identical commands, especially if we do not add a source code comment to explain this.

```suggestion
 	    rm -r artifacts
```